### PR TITLE
New Sinhala cheat sheet

### PR DESCRIPTION
**Pull request by ar-naseef**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Sinhala Cheat Sheet`**

###### Description of changes

creates Sinhala cheat sheet

###### People to notify (@abadojack @jophab)

---

Instant Answer Page: https://duck.co/ia/view/sinhala_cheat_sheets

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @ar-naseef

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ar-naseef Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by edgesince84** [comment]
Thanks @ar-naseef - feel free to share your PR with the community and ask for language review from native speakers!

File: share/goodie/cheat_sheets/json/language/sinhala.json [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+        "val": "ඔව්",
+        "trn": "ow"
+      },
+      {
+        "key": "No",
+        "val": "නැහැ",
+        "trn": "nea-he"
+      },
+      {
+        "key": "thank you",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/sinhala.json]: Lets make the capitalization of first letter of the val field consistent

File: share/goodie/cheat_sheets/json/language/sinhala.json [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+        "val": "හමුවුන එක ලොකු සතුටක්",
+        "trn": "hambu una eka loku sathutak"
+      },
+      {
+        "key": "How are you?",
+        "val": "කොහොමද ඉතිං‍",
+        "trn": "kohomadha. Ithin?"
+      },
+      {
+        "key": "good",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/language/sinhala.json]: Lets make other key fields' first letter capital too.

Comment (ar-naseef) [inline share/goodie/cheat_sheets/json/language/sinhala.json]: @jophab done!

**Comment by ar-naseef** [comment]
Can anybody tell me, how to proceed or what should I do next. I am new here. This is my first cheat sheet

**Comment by jophab** [comment]
Your cheat sheet will be merged after all modifications are made. I think this good to merge now.
@tagawa @GuiltyDolphin 
Then it will go live after an internal review

**Comment by ar-naseef** [comment]
I have no idea why it takes too long, its been a month since my made this pull request.

**Comment by GuiltyDolphin** [comment]
@ar-naseef Sorry for the delay, we're currently waiting on review from native speakers. Are you a native speaker yourself?

cc @tagawa @moollaza

**Comment by ar-naseef** [comment]
@GuiltyDolphin yeah, I am a native speaker. And I m a Sri Lanakn.

**Comment by ar-naseef** [comment]
@moollaza @GuiltyDolphin is there anything I can do here to make the process faster.

**Comment by moollaza** [comment]
@ar-naseef we're technically no longer merging Pull Requests that are unrelated to our current [Programming Mission](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53) but because this is a cheat sheet, and it was started quite a while ago, I'll merge it.

Would you be interested in getting involved with our new mission, to make DuckDuckHack the best search engine for programmers?

If so I suggest you join the DuckDuckHack Forum and tackle some Programming Mission tasks 👍

**Pull request merged**